protocols/295c73/295c73.ot2.apiv2.py

```diff
@@ -0,0 +1,381 @@
+     output_sample_col, mag_sample_col, mag_post_index_col
+     ] = get_values(  # noqa: F821
+      'sample_count', 'hold_pcr_plate_on_ice_until_block_reaches_98',
+      'sample_volume', 'mm_volume', 'room_temp', 'mm_well', 'beads_well',
+      'index_rxn_mx_well', 'PEG_NaCl_well', 'post_PCR_TE_well',
+      'initial_sample_col', 'indexing_sample_col', 'output_sample_col',
+      'mag_sample_col', 'mag_post_index_col')
+
+    ctx.set_rail_lights(True)
+
+    # turn off rail lights to bring the pause to the user's attention
+    ctx.set_rail_lights(False)
+    ctx.pause("""Please pre-cool both the thermocycler block and the
+    temperature module to 4 degrees via settings in the Opentrons app prior to
+    running this protocol. Please add multiplex master mix and PCR plate
+    containing 10 ul DNA samples to their locations on the OT-2 deck""")
+    ctx.set_rail_lights(True)
+
+    # setup p20 single channel, p300 single channel, tips
+    tips20 = [ctx.load_labware("opentrons_96_filtertiprack_20ul", '5')]
+    tips300 = [ctx.load_labware("opentrons_96_tiprack_300ul", '9')]
+    p20 = ctx.load_instrument(
+        "p20_single_gen2", 'left', tip_racks=tips20)
+    p300 = ctx.load_instrument(
+        "p300_single_gen2", 'right', tip_racks=tips300)
+
+    # setup temperature module at 4 degrees for multiplex mastermix
+    temp = ctx.load_module('Temperature Module', '3')
+    temp_reagents = temp.load_labware(
+        'opentrons_24_aluminumblock_nest_0.5ml_screwcap',
+        'Opentrons 24-Well Aluminum Block')
+    temp.set_temperature(4)
+
+    # setup 4 degree thermocycler for mastermix addition to initial samples
+    tc = ctx.load_module('thermocycler')
+    tc.open_lid()
+    tc_plate = tc.load_labware("biorad_96_wellplate_200ul_pcr")
+    tc.set_block_temperature(4)
+
+    # setup magnetic module with status 'disengaged'
+    mag = ctx.load_module('magnetic module gen2', 6)
+    mag_plate = mag.load_labware("nest_96_wellplate_100ul_pcr_full_skirt")
+    mag.disengage()
+
+    # define initial DNA sample locations
+    # avoid wells located at plate edge to minimize evaporation during PCR
+    # example B2 to G2, B3 to G3, B4 to B4, B5 to G5 for 24 samples
+    initial_sample = [well for column in tc_plate.columns()[
+      initial_sample_col:] for well in column[1:len(column)-1]][:sample_count]
+
+    # setup samples, indices, ETOH, master mix, beads, waste, output
+    column_well_count = len(tc_plate.columns()[0])
+    indexing_sample = tc_plate.wells()[
+      indexing_sample_col*column_well_count:sample_count +
+      indexing_sample_col*column_well_count]
+    output_sample = tc_plate.wells()[
+      output_sample_col*column_well_count:sample_count +
+      output_sample_col*column_well_count]
+    multiplex_mm = temp_reagents.wells_by_name()[mm_well]
+    index = ctx.load_labware("biorad_96_wellplate_200ul_pcr", 4)
+    reservoir = ctx.load_labware("nest_12_reservoir_15ml", '2')
+    ETOH = reservoir.wells()[0]
```

Comment:
generally, we reserve all capital variables for global variables. Let's change this to `etoh`